catalog.html

```diff
@@ -62,7 +63,7 @@ <h1 class="inner-header-title">Гостиницы Седоны</h1>
       </header>
 
       <section class="catalog-products">
-        <h2 class="visually-hidden">Список товаров с фильтрами</h2>
+        <h2 class="visually-hidden" hidden>Список товаров с фильтрами</h2>
```

Comment:
```suggestion
        <h2 class="visually-hidden">Список товаров с фильтрами</h2>
```
хммм, это точно тут нужно?